### ISSUE TITLE
Can't emboss most materials of bow limbs onto bows (short, long, and cross)

### ISSUE THREAD

**Issue by nighthawk475** [SlimeKnights/TinkersConstruct#3122]
I believe I've found a bug involving embossing bow limbs as I can't see a way this is intended.

More than half of the bow limbs are unable to be embossed onto a bow (shortbow longbow or crossbow, the bomb limbs which do not work are the same for all three), specifically I've realized it's any bow limb made from a material which has two different traits for the head and extra.

I originally encounter this with my own custom mod pack on TiC version TConstruct-1.12-2.7.2.15
I have since also tested TConstruct-1.12-2.7.2.19 both in the same modpack and entirely on it's own (only with the mantle dependency in my mods folder).

The bow limbs that do not work for embossing are all of the bow limbs with two 'traits'/effects (one for the head one for the bow/extra.

For example, no shortbow can be embossed with cobalt bow limbs, which has Lightweight and Momentum
meaning it is impossible to give a shortbow the lightweight draw speed improvement.
the same goes for all of the other materials with two different effects (even Paper, which has two separate writable traits, cannot be embossed onto a shortbow,)

The ones that do work are ones like Obsidian or Wood which have a single effect regardless of part type "Duritae" and "Ecological" respectively.

This is most heavily impacting on the shortbow as there is no other part type to emboss these materials with. where as the longbow still allows you to emboss them as plates and the crossbow as crossbow as tough bindings/toolrods.

Thank you for your time, I hope this is helpful and something that can be resolved.

Edit: I should have looked for this beforehand, but I've also found a previous issue submition that's already been closed for failure to reproduce: https://github.com/SlimeKnights/TinkersConstruct/issues/2988
I believe the reasons I've outlined above about it only affecting some materials are still the cause, and the picture in that thread does use one of the materials I noted as working.

**Comment by bonii-xx** [comment]
>specifically I've realized it's any bow limb made from a material which two different traits for the head and extra.

That sounds like a lead. Will investigate.

**Comment by nighthawk475** [comment]
Thanks! This has really been limiting my options for bow crafting, I've found it also impacting my optimal choices for long bows and repeaters too, given that more than 50% of the materials can't be used for their head effect on a bow embossment now, and from the look of the other thread I found this bug has probably been around for a while. 

If there's anything else I can test or any logs I can provide to help you reproduce/fix this I'd be glad to. Just let me know.
